libpod/pod.go

```diff
@@ -70,6 +71,10 @@ type PodConfig struct {
 
 	// ID of the pod's lock
 	LockID uint32 `json:"lockID"`
+	//
```

Comment:
We need comments for the documentation not empty comments.

Reply:
Sorry, I must have missed some comments before committing, should be fixed now.